vlib/v/parser/tests/struct_field_unknown_module_b.out

```diff
@@ -0,0 +1,5 @@
+vlib/v/parser/tests/struct_field_unknown_module_b.vv:2:7: error: unknown module `term.ui`
```

Comment:
I think this error might be misleading (the `term.ui` module _does_ exist, but the syntax to access it is wrong)

Reply:
True.

Reply:
That's right. But I think parser cannot detect whatever module is exists. Searching module logic is in builder

Reply:
No, it's not. Look for `util.qualify_import` in the parser

Reply:
`qualify_import` returns argument mod as is. So it cannot detect whether module is found or not, can it?

 I think It is better to move `Builder.find_module_path` into `v.util` then use it. But it affects a lot of places,

And this error message is not added by this PR. I just add a test. If you use `term` without import, V says `unknown module`  It should be improved to more clear message, but I think another PR would be better.

Reply:
The error is fine, it is for a .vv file that *did not* `import term.ui`.

Reply:
@zakuro9715 however, you can perhaps use a different nonexisting name like `unknownmodule.unknownsubmodule.Code`, so that there is no confusion